libs/core/algorithms/include/hpx/parallel/unseq/loop.hpp

```diff
@@ -26,12 +26,12 @@ namespace hpx::parallel::util {
         struct unseq_loop_n
         {
             template <typename InIter, typename F>
-            HPX_HOST_DEVICE HPX_FORCEINLINE static constexpr InIter call(
+            HPX_HOST_DEVICE HPX_FORCEINLINE static InIter call(
                 InIter HPX_RESTRICT it, std::size_t num, F&& f)
             {
                 // clang-format off
-                HPX_IVDEP HPX_UNROLL HPX_VECTORIZE
-                for (std::size_t i = 0; i != num; ++i)
+                HPX_VECTORIZE
```

Comment:
Uhh, does this PR completely remove vectorization for non-omp based schemes?

Reply:
I believe HPX_VECTORIZE does vectorize is absence of OMP too, it defaults to compiler supported vectorization

Reply:
Ok, we need to proof of that statement. Also, if this is the case, can we remove the other macros?